package-scripts.yml

```diff
@@ -0,0 +1,48 @@
+scripts:
```

Comment:
Interesting. Does this PR add a `package-scripts.yml` in addition to the existing `package-scripts.js` file?